api/module.mk

```diff
@@ -35,12 +35,20 @@ $(BUILD): $(SCM_SRC) $(SCM_RESOURCES)
 	$(LN) $(JAR) $(JAR_LINK)
 	@$(TOUCH) $@
 
-LOGFILE := $(DERIVED_MODULE_DIR)/test.log
-$(CHECK): $(SCM_SRC) $(SCM_RESOURCES) $(CLOJURE_PROJECT)
+$(UNIT): $(SCM_SRC) $(SCM_RESOURCES) $(CLOJURE_PROJECT)
 	$(EXPORT) CPCACHE=$(CPCACHE_DIR);                        \
-	$(CLOJURE) $(CLJFLAGS) -A:test unit | $(TEE) $(LOGFILE)
+	$(CLOJURE) $(CLJFLAGS) -A:test unit | \
+	$(TEE) $(DERIVED_MODULE_DIR)/unit.log
 	@$(TOUCH) $@
 
+$(INTEGRATION): $(SCM_SRC) $(SCM_RESOURCES) $(CLOJURE_PROJECT)
+	$(EXPORT) CPCACHE=$(CPCACHE_DIR);                        \
+	$(CLOJURE) $(CLJFLAGS) -A:test integration | \
+	$(TEE) $(DERIVED_MODULE_DIR)/integration.log
+	@$(TOUCH) $@
+
+$(CHECK): $(UNIT) $(INTEGRATION)
```

Comment:
I'd move this into  `modules.mk` to share with all modules

Reply:
If you don't indent to execute any steps in `check` (i.e. it's just an alias for `unit` and `integration`) you don't need to worry about the CHECK timestamp. Simply add the following after `makerules/modules.mk:35`:
```
check: unit integration
```